client/sections.js

```diff
@@ -85,11 +79,23 @@ const sections = [
 		group: 'me',
 	},
 	{
```

Comment:
Just like we've explained why for `reader` the primary section that handles `/read` has to be first, would you mind adding a comment to this one, explaining why it's at the bottom?

Also, unrelated, but after this change it might make sense to handle `/me/*` and redirect it  to `/me` in this section.